spec/mongoid/extensions/time_spec.rb

```diff
@@ -166,113 +166,316 @@
 
     context "when the value is a string" do
 
-      context "when using active support's time zone" do
-        include_context 'using AS time zone'
+      context "when use_utc is false" do
+        config_override :use_utc, false
 
-        context "when the string is a valid time with time zone" do
+        context "when using active support's time zone" do
+          include_context 'using AS time zone'
 
-          let(:string) do
-            # JST is +0900
-            "2010-11-19 00:24:49.123457 +1100"
+          context "when the string is a valid time with time zone" do
+
+            let(:string) do
+              # JST is +0900
+              "2010-11-19 00:24:49.123457 +1100"
+            end
+
+            let(:mongoized) do
+              Time.demongoize(string)
+            end
+
+            let(:expected_time) { Time.parse("2010-11-18 13:24:49.123457 +0000").in_time_zone }
```

Comment:
This would be more clear if the time zone was given explicitly in the construction call.